src/Employer/UnitTests/Employer.Web/ViewModels/ApplicationReview/ApplicationReviewEditModelTests.cs

```diff
@@ -89,11 +96,39 @@ public void ShouldBeInvalid()
                 CandidateFeedback = "?$@#()\"\'\\!,+-=_:;.&€£*%/[] \\A-Z \a-z \0-9 your comments will be sent to the candidate."
             };
 
-            var validator = new ApplicationReviewEditModelValidator();
+            var validator = new ApplicationReviewEditModelValidator(MockProfanityListProvider);
 
             var result = validator.Validate(m);
 
             result.IsValid.Should().BeFalse();
         }
+
+        [Theory]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "Some candidate feedback bother")]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "dang Some candidate feedback")]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "Some candidate balderdash bother")]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "Some drat feedback bother")]

```

Comment:
Add a success test as well

Reply:
Success scenarios are covered in the earlier tests.